src/display/api.js

```diff
@@ -834,7 +841,8 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
                                                       this.objs,
                                                       this.commonObjs,
                                                       intentState.operatorList,
-                                                      this.pageNumber);
+                                                      this.pageNumber,
+                                                      this.canvasFactory);
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99022111, this line should be replaced by `canvasFactory`, with
```javascript
var canvasFactory = params.canvasFactory || new DOMCanvasFactory();

```
defined at line https://github.com/mozilla/pdf.js/pull/8002/files#diff-0ecad279ed2252e3eb47a4d96ec1463cR815 above.